release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -3153,14 +3153,33 @@ $ oc adm release info 4.11.22 --pullspecs
 
 * Any 4.11 and later  `arm64` cluster that has more than around 470 containers to a node, can cause additional pod creation to fail with the following error:
 +
-[source,terminal]
-====
+[source,text]
```

Comment:
This should stay [source,terminal], everything else looks good

Reply:
:heavy_check_mark: